### DISCUSSION TITLE
practica css, no entender los bonus

### DISCUSSION THREAD

**Discussion by damipere** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#421] [Q&A]
Bonus de diseños o páginas adicionales (Totalmente a tu creatividad)
Bonus de despliegue de la página
a que se refiere a despliegue de la pagina?? y las paginas adicionales son paginas que hice anteriormente?

**Comment by carlos-gmx3** [comment]
se refiere a que subas tu pagina a un servicio de host gratuito como netlify entre otros, hasta lo puedes hacer aquí mismo, en github, solo compartes el link para que todos las puedan ver.
Paginas adicionales es que hagas más paginas para el sitio de vacunas, que si presionas uno de los links de la página clon te lleve a más paginas que te inventes. Recuerdas los wire frames? que diseñabas las múltiples paginas y vistas que el link de una te llevaba a otra y así? pues es eso.

**Comment by MauroMontan** [comment]
Puedes hacer un host muy facil aquí mismo en github, usando tu repositortio


- Primero tienes que irte a la parte de settings de tu repositorio

![image](https://user-images.githubusercontent.com/68311575/158671111-049ee232-8601-47b6-989d-eaeb9abae59f.png)

- Bajaras hasta el apartado de github pages

![image](https://user-images.githubusercontent.com/68311575/158671794-38c8f1bc-6e85-4cbe-ba37-32346b9da41e.png)

- Tendras que especificar la rama y colocar en folder (/root), das en guardar y esperaras menos de 5 minutos para poder visualizar tu pagina

![image](https://user-images.githubusercontent.com/68311575/158673648-de49307e-7273-4cf6-95ae-f6d6a93f675b.png)

- Al final te dara un link, en donde tu pagina está hostedada

![image](https://user-images.githubusercontent.com/68311575/158674286-40e58d03-382f-4731-8515-707d577c6665.png)


espero que te sirva, si tienes alguna duda puedes consusltar en la documentación [aquí](https://pages.github.com/)
o revisar un tutorial mas detallado.
